src/ai/backend/manager/models/gql_models/agent.py

```diff
@@ -689,7 +689,7 @@ def from_row(
         cls,
         ctx: GraphQueryContext,
         row: Row,
-    ) -> Agent:
+    ) -> AgentSummary:
```

Comment:
It is now better to use [`typing.Self`](https://docs.python.org/3/library/typing.html#typing.Self).